src/main/webapp/app/exam/manage/student-exams/student-exam-detail-table-row/student-exam-detail-table-row.component.ts

```diff
@@ -72,4 +70,31 @@ export class StudentExamDetailTableRowComponent implements OnChanges {
         }
         return route;
     }
+
+    getBonusPoints(exercise: Exercise): number | undefined {
+        if (!exercise) {
+            return 0;
+        }
+        switch (exercise.includedInOverallScore) {
+            case IncludedInOverallScore.INCLUDED_COMPLETELY:
+                return exercise.bonusPoints;
+            case IncludedInOverallScore.INCLUDED_AS_BONUS:
+                return exercise.maxPoints;
+            default:
+                return 0;
+        }
+    }
+
+    getMaxPoints(exercise: Exercise): number | undefined {
+        if (!exercise) {
+            return 0;
+        }
+        switch (exercise.includedInOverallScore) {
+            case IncludedInOverallScore.INCLUDED_COMPLETELY:
+                return exercise.maxPoints;
+            case IncludedInOverallScore.INCLUDED_AS_BONUS:
```

Comment:
```suggestion
```
The explicit fall-through case can be removed.
As we only care about a single condition here, an if/else would do the job as well (personal preference).

Reply:
I would prefer to keep it that way to make it consistent with the `getBonusPoints` method above (because they are somewhat complementary) and to make extending it easier if we ever add new values to the underlying enum if that is okay for you as well.